app/scripts/filters/resources.js

```diff
@@ -120,16 +120,29 @@ angular.module('openshiftConsole')
       });
       return nameCount;
     }
-    return function (resource, projects){
+    function escapeLessThanGreaterThan(str) {
+      return str.
+        replace(/</g, '&lt;').
+        replace(/>/g, '&gt;');
+    }
```

Comment:
I explicitly opted for this custom function so I could scope it to only < and > (`_.escape` covers &, <, >, ", ', and \`).   Given &, ", ', and \` don't  cause the bug, I thought it was better to leave them out of the escaping so search highlighting isn't broken for them, too.  Yea/nay?